docs/New-MarkdownHelp.md

```diff
@@ -5,60 +5,58 @@ schema: 2.0.0
 
 # New-MarkdownHelp
 ## SYNOPSIS
-Convert your existing external help into markdown or generate it from Help object.
+Creates help in markdown format.
+
 ## SYNTAX
 
-### FromModule
+### UNNAMED_PARAMETER_SET_1
```

Comment:
I remember that - I should have synced again before editing. I hope that doesn't add a lot of churn to your review.